src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/DependencyInjection/MvcJsonMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extensions methods for configuring MVC via an <see cref="IMvcBuilder"/>.
+    /// </summary>
+    public static class MvcJsonMvcBuilderExtensions
+    {
+        /// <summary>
+        /// Configures JSON.NET specific input and output formatters.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        public static IMvcBuilder AddNewtonsoftJsonFormatters(this IMvcBuilder builder)
```

Comment:
Thoughts on using a distinct name here? `IMvcBuilder` did not previously have Json extensions, so these aren't breaking although I did change the extensions on `IMvcCoreBuilder` to match these names

Reply:
Sounds right to me.